app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -111,11 +111,12 @@ function useUspsLocations() {
   }, []);
 
   // sends the raw text query to arcgis
-  const { data: addressCandidates, isLoading: isLoadingCandidates } = useSWR([addressQuery], () =>
-    addressQuery ? requestAddressCandidates(addressQuery) : null,
-  );
+  const {
+    data: addressCandidates,
+    isLoading: isLoadingCandidates,
+    error: addressError,
```

Comment:
very small thing - but is it clearer if we change the name of the err to addressCandidatesError?

Reply:
I decided to keep `addressError` for brevity's sake.  Probably either variable name would be fine. 🤷🏻‍♀️